### ISSUE TITLE
Allow other tag types to be discovered.

### ISSUE THREAD

**Issue by danaki** [Ultz/XNFC#1]
Hello,

I'm a newbie in Xamarin development. I've installed your lib and tried to deploy XnfcTest application on a real Huawei P10 device. I had to disable system "Tags" application and then after pressing "Scan" button nothing happens. My NFC-card is a "Mifare Ultralight" public transportation card and I tested it with an "NFC reader" taken from Appstore and it works. However, XNFC test application doesn't catch it at all (nothing happens and no message in debug console). I can guess that TECH_DISCOVERED must be configured as well like it [described here](https://stackoverflow.com/questions/9211743/android-new-tag-collected-unknown-tag-type)? Can you help me with this issue?

**Comment by Perksey** [comment]
Hi Yuri. XNFC currently only supports NDEF as far as I know, so you probably have a card that doesn't use NDEF. Let me know the format of your card to see if this is a bug or not; I do plan to rewrite XNFC at some point to support a wide range of formats.

**Comment by danaki** [comment]
I'm not sure l, can you look at the attached screenshot? Is this format
supported?

вс, 3 мар. 2019 г., 15:48 Dylan Perks <notifications@github.com>:

**Comment by Perksey** [comment]
You haven't attached a screenshot.

**Comment by danaki** [comment]
Yes I see, figuring out how to attach it

вс, 3 мар. 2019 г., 15:57 Dylan Perks <notifications@github.com>:

**Comment by danaki** [comment]
![screenshot_20190303-155336](https://user-images.githubusercontent.com/1043805/53696208-86c29580-3dcd-11e9-80b5-ad7f7afa1e6f.jpg)

**Comment by Perksey** [comment]
Yep, that's not an NDEF tag. You can format this tag to NDEF if you wish, that will make XNFC able to pick it up.

We originally wrote XNFC as a proprietary piece of software for one of our products, thus compatibility and other uses weren't at the top of our list. We'll mark this as a feature request for a future version.
